### ISSUE TITLE
xformers error

### ISSUE THREAD

**Issue by Gyro0o** [TheLastBen/fast-stable-diffusion#776]
![bandicam 2022-11-30 23-46-00-631](https://user-images.githubusercontent.com/117460296/204931027-268e3a10-cab6-4af6-b546-d20a0c02bf7e.png)
When I run the cell I get this error

**Comment by Gyro0o** [comment]
Update:
it seems that
pip install wget
did fix it

**Comment by Gyro0o** [comment]
![bandicam 2022-12-01 00-48-15-950](https://user-images.githubusercontent.com/117460296/204932398-e6ba3585-b95c-4282-91a0-e9d96df185af.png)

Now I got this...

**Comment by illumnat** [comment]
> ![bandicam 2022-12-01 00-48-15-950](https://user-images.githubusercontent.com/117460296/204932398-e6ba3585-b95c-4282-91a0-e9d96df185af.png)
> 
> Now I got this...

Yeah... I had the wget thing too but got that installed. Now I'm getting the same error as your screenshot.

**Comment by fragloser** [comment]
Same error  "no module named 'wget'

**Comment by GabrielMirandaOliveira** [comment]
para arrumar o erro do wged, troque a parte de cima do codio para:

from subprocess import getoutput
from IPython.display import HTML
from IPython.display import clear_output
!pip install wget
import wget
import time

**Comment by GabrielMirandaOliveira** [comment]
![image](https://user-images.githubusercontent.com/30904933/204934343-fe4417e3-4e46-41c3-9e02-73c8bb8be078.png)

**Comment by TheLastBen** [comment]
will be fixed in an hour

**Comment by GabrielMirandaOliveira** [comment]
some form of contribution that accepts real (Brazilian currency)

**Comment by younyokel** [comment]
> will be fixed in an hour

![image](https://user-images.githubusercontent.com/53343081/204950503-51b8b745-7139-4e42-b860-8a10d3129827.png)
after the fix this is what i get
